tests/benchmarks/test_benchmark.py

```diff
@@ -67,6 +69,20 @@ def protostar_cairo_path_fixture(request) -> Path:
     return (Path(request.node.fspath).parent.parent.parent / "cairo").absolute()
 
 
+def get_test_project_compiler(
+    include_paths: Optional[List[str]] = None,
+) -> ProjectCompiler:
+    # FIXME(arcticae): Build a project compiler with fake protostar.toml in the future if those tests are needed
+    return None  # type: ignore
+    # return ProjectCompiler(
```

Comment:
Put this beside `FIXME` comment, indented, so that TODO browsers (for example IntelliJ's one) can see this snippet.

Reply:
i deleted benchmarking afterall, seems a bit silly to keep maintaining it, while it's unused.